core/resources/service-config/demo-config.yml

```diff
@@ -30,7 +30,5 @@ app:
   debug: False
   messaging:
     url: "nats://localhost:4222"
-    stream_name: healthos
```

Comment:
these settings are no longer needed as we are defining these fields using the typing system's "Literal" type with a Pydantic default value.

```python
stream_name: Literal["healthos"] = "healthos"
```